numba/targets/gdb_hook.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import print_function, absolute_import
+
+import os
+import sys
+
+from llvmlite import ir
+from llvmlite.llvmpy.core import Constant, Type
```

Comment:
We should try not to add new usage of `llvmlite.llvmpy`, which only existed for temporary backend compatbility.